framework/tlvf/yaml/tlvf/test/tlvVarList.yaml

```diff
@@ -38,3 +38,7 @@ cInner:
     _type: uint8_t
     _length: [ list_length ]
   var1: uint32_t
+  unknown_length_list:
+    _type: uint8_t
+    _length: []
```

Comment:
This cannot possibly be parsed. An unknown length list can only exist as the last field in a TLV, because it uses the TLV length to determine the length of the list. Here, cInner is used in e.g. the outer class's var1, so there is no way where the boundary lies between var1's unknown_length_list and the outer unkown_length_list. (Note the confusion with the names :-).